vermouth/processors/make_bonds.py

```diff
@@ -76,25 +80,36 @@ def bonds_from_distance(system, fudge=1.1):
         certain distance from each other. It is probably disconnected.
     """
     system = nx.compose_all(system.molecules)
-    idx_to_nodenum = {idx: n for idx, n in enumerate(system)}
+    idx_to_nodenum = {
+        idx: n
+        for idx, n in enumerate(
+                subn
+                for subn in system
+                if system.nodes[subn].get('element') in VDW_RADII
+        )
+    }
     max_dist = max(VDW_RADII.get(node.get('element'), 0.2) for node in system.nodes.values())
```

Comment:
Why `0.2` and not, say, `max(VDW_RADII.values())`? And if elements not in VDW_RADII don't make bonds, this seems rather pointless anyway. Just take the largest radius you find for the elements we know: those in `idx_to_nodenum`